files/build_templates/startup_tsa_tsb.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description= STARTUP TSA-TSB SERVICE
+Requires=updategraph.service database.service
+After=updategraph.service database.service
+ConditionPathExists=!/etc/sonic/chassisdb.conf
+
+[Service]
+Environment="STARTED_BY_TSA_TSB_SERVICE=1"
```

Comment:
after the service is stopped do we need to reset `STARTED_BY_TSA_TSB_SERVICE`

Reply:
Reset the env variable